public/app/features/explore/Panel.tsx

```diff
@@ -4,18 +4,27 @@ interface Props {
   isOpen: boolean;
   label: string;
   loading?: boolean;
-  onToggle: (isOpen: boolean) => void;
+  isCollapsable?: boolean;
```

Comment:
Spelling issue, also I'd propose `collapsible`, it's shorter. Or is there a benefit with the `is` prefix?

Reply:
Good catch. I went for is since `isOpen` was used. Will change :+1: